src/pepr/operator/crd/sources/exemption/v1alpha1.ts

```diff
@@ -0,0 +1,99 @@
+import { V1CustomResourceDefinitionVersion, V1JSONSchemaProps } from "@kubernetes/client-node";
+
+export const v1alpha1: V1CustomResourceDefinitionVersion = {
+  name: "v1alpha1",
+  served: true,
+  storage: true,
+  additionalPrinterColumns: [
+    {
+      name: "Status",
+      type: "string",
+      description: "The status of the exemption",
+      jsonPath: ".status.phase",
+    },
+    {
+      name: "Age",
+      type: "date",
+      description: "The age of the exemption",
+      jsonPath: ".metadata.creationTimestamp",
+    },
+  ],
+  subresources: {
+    status: {},
+  },
+  schema: {
+    openAPIV3Schema: {
+      type: "object",
+      properties: {
+        status: {
+          type: "object",
+          properties: {
+            observedGeneration: {
+              type: "integer",
+            },
+            phase: {
+              enum: ["Pending", "Ready", "Failed"],
+              type: "string",
+            },
+          },
+        } as V1JSONSchemaProps,
+        spec: {
+          type: "object",
+          properties: {
+            exemptions: {
+              type: "array",
+              description: "Policy exemptions",
+              items: {
+                type: "object",
+                required: ["policies", "matcher"],
+                properties: {
+                  description: {
+                    type: "string",
+                    description:
```

Comment:
I think we should probably do something a little different here than Pkg. Probably should make this `title` and include a freetext `description` prop as well so we can include the exemption data in the CR itself and it won't be lost when the YAML/JSON is processed. This will be important with UDS Engine to show why something was exempted. This would also mean all the various CR YAML comments we currently have in this PR should move to this description field. I'd also keep at least `title`, but maybe also `description` in the CR status table.

Reply:
Oh yeah, i'd totally forgotten about the description for the name. i like that, making the description as the reasons for the exemption. Was chatting with @mjnagel and we were wondering about the `title`. Since we don't necessarily need that for processing and storing the exemptions, would that be more for runtime engine and reporting?